access/ms-teams/_tpl/teleport-ms-teams.toml

```diff
@@ -0,0 +1,14 @@
+preload = false
+
+[teleport]
+addr = "localhost:3025"
+identity = "identity"
```

Comment:
Can we add some comments explaining how to obtain the `identity` file?
Similar to what we have in Slack